docs/zh_CN/Release.md

```diff
@@ -38,7 +173,7 @@
 #### 训练平台改进
 
 * 改进 OpenPAI YAML 的合并逻辑
-* 支持将 Windows 作为[远程模式](https://github.com/microsoft/nni/blob/master/docs/zh_CN/TrainingService/RemoteMachineMode.md#windows)中的计算节点
+* 支持 Windows 在 Remote Mode 中作为远程机器 [Remote Mode](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/RemoteMachineMode.md#windows)
```

Comment:
Check document how to make such hidden strings visible to prevent fallback to en document.